Makefile

```diff
@@ -293,9 +294,14 @@ endif
 test-acceptance-smoke:
 	$(Q)TEST_ACCEPTANCE_TAGS=@smoke $(MAKE) test-acceptance
 
+.PHONY: test-acceptance-report
+## Generate acceptance test report
+test-acceptance-report:
+	$(Q)$(HACK_DIR)/generate-allure-report.sh
+
 .PHONY: test-acceptance-artifacts
 ## Collect artifacts from acceptance tests to be archived in CI
-test-acceptance-artifacts:
+test-acceptance-artifacts: test-acceptance-report
```

Comment:
generating reports should optional IMHO, otherwise, it is going to fail where docker is not installed.

Reply:
good point